core/src/style/pointStyle.h

```diff
@@ -50,6 +53,8 @@ class PointStyle : public Style {
 
     UniformLocation m_uTex{"u_tex"};
     UniformLocation m_uOrtho{"u_ortho"};
+
+    mutable std::unique_ptr<LabelMesh> m_mesh;
```

Comment:
Naming is one of the hard problems. I think a comment on the member would be sufficient. Do you have a suggestion? meshBuffer, labelVertexBuffer or something that hints that the content are updated each frame?